src/mmw/js/src/modeling/gwlfe/subbasin/views.js

```diff
@@ -443,6 +463,14 @@ var CatchmentsTableView = Marionette.ItemView.extend({
             rows: catchments,
             catchmentDetails: catchmentDetails,
             summaryRow: summaryRow,
+            areaUnit: coreUnits[scheme].AREA_L.name,
+            areaDisplayUnit: 'AREA_L',
+            massPerAreaUnit: coreUnits[scheme].MASSPERAREA_L.name,
+            massPerAreaDisplayUnit: 'MASSPERAREA_L',
```

Comment:
I'm seeing Loading Rates (area normalized) in `ton/ac`:

![image](https://user-images.githubusercontent.com/1430060/49237737-dce9fc00-f3cc-11e8-9219-457323dc6199.png)

But the spreadsheet asks them to be in `lb/ac`:

![image](https://user-images.githubusercontent.com/1430060/49237778-f55a1680-f3cc-11e8-9a81-9c679dd8469f.png)

Use `MASSPERAREA_M` instead of `MASSPERAREA_L`

Reply:
Thanks. Pushed this fix.